src/core.cpp

```diff
@@ -72,6 +81,10 @@ namespace Core
 	
 	/** The Address for the Last Round. **/
 	std::string LAST_ROUND_BLOCKFINDER = "2S4PPSznWfPVLtPJNpi8Ly46Wft3wGbayGkhaGzKLVcepmrhKTP";
```

Comment:
(I can see this comes from as far back as 2015 so not really part of the pull request review - but don't want to forget to ask) @VidereLicet  Should this start out blank?